tests/TestCase/Filesystem/FileTest.php

```diff
@@ -60,7 +60,12 @@ public function testParseDir(): void
         $actual = File::parseDir($dir, $defaultDomain, $translations);
         static::assertFalse($actual); // there is an empty file in dir test_dir
         static::assertNotEmpty($translations);
-        static::assertSame($expected, $translations);
+        foreach ($expected as $key => $values) {
```

Comment:
Probably `static::assertEquals($expected, $translations);` would have worked the same, and it would have failed in case of unexpected keys present in the final array.

Reply:
`assertEquals` looks better... I'll give it a try